addon/services/rollbar.js

```diff
@@ -10,11 +10,17 @@ export default Ember.Service.extend({
   }),
 
   config: Ember.computed(function() {
-    return Ember.getOwner(this).resolveRegistration('config:environment').emberRollbarClient;
```

Comment:
Why didn't you use the ember-cli-app-version? :)

Reply:
Didn't want to add an extra dependency for an addon, as well as there was a problem with the integration, so I've decided to go that way to move on.